examples/display_usb_hid/main/idf_component.yml

```
@@ -1,7 +1,7 @@
 description: BSP Display USB HID Example
 dependencies:
-  idf: ">=4.4"
-  usb_host_hid: "*"
-  esp32_s3_usb_otg:
-    version: "*"
-    override_path: "../../../bsp/esp32_s3_usb_otg"
+  esp-box-3:
+    version: '*'
+    override_path: ../../../bsp/esp-box-3
+  idf: '>=4.4'
+  usb_host_hid: '*'
```

To prevent updating to 2.x releases
```suggestion
  usb_host_hid: '^1'
```